crescent/plugin.py

```diff
@@ -44,35 +50,83 @@ def load(self, path: str, refresh: bool = False) -> Plugin:
 
         Args:
             path: The module path for the plugin.
-            refresh: Whether or not to reload the plugin.
+            refresh: Whether or not to reload the plugin and the plugin's module.
         """
 
+        if refresh:
+            old_plugin = self.plugins.pop(path)
+            old_plugin._unload()
+
```

Comment:
If the plugin doesn't exist, this will raise a KeyError. I think we should either make a better error, or check if it exists before trying to unload.

Reply:
I will make it check if it exists before it unloads. Seems easier.

Reply:
i made it raise a better error